lasagne/layers/merge.py

```diff
@@ -68,22 +70,57 @@ class ElemwiseSumLayer(MergeLayer):
     before a :class:`DenseLayer`, insert separate :class:`DenseLayer` instances
     of the same number of output units and add them up afterwards. (This avoids
     the copy operations in concatenation, but splits up the dot product.)
```

Comment:
True, it has to be rewritten for merge then. But you might as well use other merge functions to aviod concatenation.